.storybook/components/Recipes/RecipesDocumentation.tsx

```diff
@@ -0,0 +1,25 @@
+import type { ReactNode } from 'react';
```

Comment:
This whole file is just copy/pasted from the other docs files. The only change is the component name on line 16. We could split this out into a reusable component and just leave the `.stories.tsx` files for all 3.
- [Documentation](https://github.com/chanzuckerberg/edu-design-system/blob/08078155c77160222708c6b8f0ef7161006365b4/.storybook/components/Docs/Documentation.tsx)
- [GettingStartedDocumentation](https://github.com/chanzuckerberg/edu-design-system/blob/08078155c77160222708c6b8f0ef7161006365b4/.storybook/components/GettingStarted/GettingStartedDocumentation.tsx)